read_it/tests/test_home_view.py

```diff
@@ -0,0 +1,27 @@
+from django.test import Client, TestCase
+from django.urls import reverse
+
+from ..models import Book
+from .setup_data import create_new_book
+
+
+class HomeViewTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        for i in range(10):
+            create_new_book()
+
+        cls.client = Client()
+
+    def test_home_returns_200(self):
+        response = self.client.get(reverse("home"))
+        self.assertEqual(response.status_code, 200)
+
+    def test_home_context_contains_five_books(self):
+        response = self.client.get(reverse("home"))
+        self.assertEqual(len(response.context["last_published_books"]), 5)
```

Comment:
Is there any reason that you create 10 boks but only check for 5 on website?

Reply:
Yes, this test makes sure that the home page will render **only** five books, even if there are more available.